BuildTools/signMeSha.bat

```diff
@@ -1,14 +1,12 @@
 rem SPDX-FileCopyrightText: Copyright (c) 2009-2020 TRUMPF Laser GmbH, authors: C-Labs
 rem SPDX-License-Identifier: CC0-1.0
 if not exist "%~dp0\real.sign" goto noSign
-echo signing with c8f244b6856c3a16604e7c00a0dd645c15ebe05a ts: http://timestamp.digicert.com
-set "SignToolEXE=%~dp0\signtool.exe"
-if exist %SignToolEXE% goto signNow
-xcopy "%ProgramFiles(x86)%\Windows Kits\10\bin\10.0.18362.0\x64\signtool.exe" "%~dp0"
-:signNow
-echo %SignToolEXE%
-%SignToolEXE% sign /sha1 c8f244b6856c3a16604e7c00a0dd645c15ebe05a /t http://timestamp.digicert.com %1
-goto exit
+if not exist "%CDE_CODE_SIGN%\azuresecrets.bat" goto noSecrets
+echo Setting Azure Vault secrets
+call ""%CDE_CODE_SIGN%\azuresecrets.bat"
+:noSecrets
+echo Signing with Azure Vault Key
+AzureSignTool.exe sign -du "https://c-labs.com" -fd sha1 -kvu %CDE_KVU% -kvi %CDE_KVI% -kvt %CDE_KVT% -kvs %CDE_KVS% -kvc %CDE_KVC% -tr http://timestamp.digicert.com -td sha1 %1
 :noSign
```

Comment:
goto exit is missing here. Otherwise its signing AND showing "!!! could not sign"